### DISCUSSION TITLE
Welcome to eslint-plugin-jest-formatting Discussions!

### DISCUSSION THREAD

**Discussion by dangreenisrael** [dangreenisrael/eslint-plugin-jest-formatting#95] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by benkimpel** [comment]
Cool. :sunglasses:

**Reply by dangreenisrael** [reply]
Super cool!

**Comment by AshtarCodes** [comment]
Very cool plugin! I'm wondering if there is anything special that needs to be done to prevent this plugin from conflicting with the prettier formatting library?

**Reply by dangreenisrael** [reply]
We've been using it along side prettier for years without any issues. Have you had a problem?
